config/splat.hd.dra.yaml

```diff
@@ -198,9 +198,14 @@ segments:
       - [0x937B8, c, 93BDC]
       - [0x94B2C, c, 94F50]
       - [0x94F7C, c, 953A0]
-      - [0x95E88, sbss]
+      - [0x95E88, .bss, bss]
   - name: dra_data
     type: data
     start: 0x9B270
     vram:  0x8013B370
+    subsegments:
+      - [0x9B270, raw, vb_0] # D_8013B6A0
```

Comment:
So there's no gap like in `us`? Haven't done the math to see if these line up with the sizes declared in the code or not
```
    subsegments:
      - [0x9B6A0, raw, vb_0] # D_8013B6A0
      - [0xDD350, raw, vb_1] # D_8017D350
      - [0xEB4E0, raw, vb_2] # D_8018B4E0 (len 108048)
      - [0x105AF0]
      - [0x109C80, raw, vb_3] # D_801A9C80 (len 64496)
```

Reply:
TBH I am not sure? That part gave me quite the headache, so I kept bruteforcing until I got an :ok: lmao

Reply:
The lengths ought to match
```
    LOAD_VAB(0, aPbav, g_VabAddrs[0], D_8013B6A0, vab0Len);
    LOAD_VAB(1, aPbav_0, g_VabAddrs[1], D_8017D350, vab1Len);
    LOAD_VAB(2, aPbav_1, g_VabAddrs[2], D_801A9C80, vab2Len);
    LOAD_VAB(3, aPbav_2, g_VabAddrs[3], D_8018B4E0, vab3Len);
```
which differ across US and HD

Reply:
0: 0xDCF1C-0x9B270=269484
1: 0xEAFEC-0xDCF1C=57552
2: 0x10978C-0xEAFEC=1087357
3: 0x115EDC-0x10978C=51024

HD
0: 269488
1: 57808
2: 65200
3: 107792

I think I did the math wrong but some of these don't seem right

Reply:
I will double-check

Reply:
The problem is that the VABs  data between HD and US do not match. The only portion of data I could find matching is `vb_0`, where in HD starts 4 bytes earlier than in HD and it's not fixable at the moment, probably due to the alignment forced being each 16 bytes.

The addresses match though, despite the data length not matching the code. I am really confused on what's happening here.

Reply:
It's expected that they aren't going to match because they replaced the voice acting for Alucard/Richter and the servants

Reply:
Shall we merge it as it is? The HD build is not even building on PC and we can fix this later if we find a solution. But I do not think I have enough knowledge to understand what's going on unfortunately

Reply:
Put a `TODO lengths are wrong. see vab1Len etc.` or something?